test/UnitTests/MSTestAdapter.UnitTests/Extensions/MethodInfoExtensionsTests.cs

```diff
@@ -244,28 +244,6 @@ public void HasCorrectTestMethodSignatureShouldReturnFalseForAsyncTestMethodsWit
 
     #endregion
 
-    #region HasCorrectTimeout tests
-
-    public void HasCorrectTimeoutShouldReturnFalseForMethodsWithoutTimeoutAttribute()
-    {
-        MethodInfo methodInfo = typeof(DummyTestClass).GetMethod("PublicMethod");
-        Verify(!methodInfo.HasCorrectTimeout());
-    }
-
-    public void HasCorrectTimeoutShouldReturnFalseForMethodsWithInvalidTimeoutAttribute()
-    {
-        MethodInfo methodInfo = typeof(DummyTestClass).GetMethod("PublicMethodWithInvalidTimeout");
-        Verify(!methodInfo.HasCorrectTimeout());
-    }
-
-    public void HasCorrectTimeoutShouldReturnTrueForMethodsWithTimeoutAttribute()
-    {
-        MethodInfo methodInfo = typeof(DummyTestClass).GetMethod("PublicMethodWithTimeout");
-        Verify(methodInfo.HasCorrectTimeout());
-    }
```

Comment:
These tests should probably be replaced by:

```csharp
    public void HasCorrectTimeoutShouldReturnFalseForMethodsWithInvalidTimeoutAttribute()
    {
        var timeoutAttribute = new TimeoutAttribute(-11);
        Verify(!timeoutAttribute.HasCorrectTimeout);
    }

    public void HasCorrectTimeoutShouldReturnTrueForMethodsWithTimeoutAttribute()
    {
var timeoutAttribute = new TimeoutAttribute(11);
        Verify(timeoutAttribute.HasCorrectTimeout);
    }
```

Also, now `PublicMethodWithInvalidTimeout` and `PublicMethodWithTimeout` are unused I think

Reply:
Looks like it. Can you add the code please @SimonCropp ?